Updating documentation (#874)
fewlinesco/connect-account: README.adoc

| `LIGHTSTEP_ACCESS_TOKEN`
| Your Lightstep access token.

| `FWL_TRACING_COLLECTORS`
| Config of the collectors used for tracing purposes.

| `SENTRY_AUTH_TOKEN`
| Auth token required to contact Sentry.

| `NEXT_PUBLIC_SENTRY_ENVIRONMENT`
| Should be set to `development`. Used to prevent sentry report when working with a built version, locally.

| `MEMCACHED_CLIENT_PASSWORD`
| Password used to initiate the Memcached Client for rate limiting.

| `MEMCACHED_CLIENT_USERNAME`
| Username used to initiate the Memcached Client for rate limiting.

| `MEMCACHED_CLIENT_SERVERS`
| Address of the Memcached Client servers. 

| `CONNECT_TEST_ACCOUNT_EMAIL`
| Email of the Connect account that will be used for e2e tests.

| `CONNECT_TEST_ACCOUNT_PASSWORD`
| Password of the Connect account that will be used for e2e tests.

| `CONNECT_TEST_ACCOUNT_SUB`
| Sub of the Connect account that will be used for e2e tests.

| `CONNECT_TEST_ACCOUNT_URL`
| URL of the Connect Account application that will be used for e2e tests.

> 

Reviewer: Could you renamed the `Connect account` mentions with `Connect.Account`? 🙇‍♂️.

- *configs/*: 
  * *config-variables.ts*: Entry point used to verify env vars sourcing, and prevent the app to run if forgotten.
  * *db-client.ts*: Singleton of the DynamoDB client.

>   * *intl.ts*: 

Author: @Fuzznimp 😬

Reviewer: > File regrouping all the `intl` object, which are instances to store the cache of all Intl.* APIs, configurations, compiled messages and such.

- *app.json*: Building instructions for Heroku.
- *assets.d.ts*: Type declaration allowing the import of assets in TypeScript files.
- *Brewfile*: Tools needed that will be installed via Brew for MacOS users.
- *connect-profile-openapi.yml*: OpenAPI file of Connect Profile.
- *docker-compose.yml*: Instructions to launch DynamoDB and the observability tools.

> - *Dockerfile*: Instructions for Docker image build process.
> - *lighthouserc.js*: Instructions for Lighthouse reviews.

Reviewer: ```suggestion
- *Dockerfile*: Instructions for Docker image build process.
- *lighthouserc.js*: Instructions for Lighthouse audits.
```

We are using https://github.com/getgauge/taiko[Taiko] to test our workflows.

=== CI/CD pipeline

To automate our test processes, we use https://circleci.com[CircleCI], which allow us to run our test suites on new commit pushed to each pull request.

Several flows (or `jobs`) are executed in this context and orchestrated in workflows inside the `config.yml` file: 

> - *lint-and-tests*: runs code linting and integration/unit tests.
> - *e2e-tests*: runs e2e tests suite on a deployed Connect Account version.
> - *lighthouse*: runs Lighthouse reports on a deployed Connect Account version.
> - *promote-to-production*: use to automate production deployment from staging when all tests suite is successful.

Reviewer: ```suggestion
- *lint-and-tests*: runs the code linting and the integration/unit tests.
- *e2e-tests*: runs the e2e test suite on a deployed Connect Account version.
- *lighthouse*: runs the Lighthouse audits on a deployed Connect Account version.
- *promote-to-production*: used to automate the production deployment from staging when the test suite is successful.
```
